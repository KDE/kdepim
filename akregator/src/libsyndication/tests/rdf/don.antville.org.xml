<?xml version="1.0" encoding="iso-8859-1" standalone="yes"?>
<rdf:RDF 
  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
  xmlns:dc="http://purl.org/dc/elements/1.1/"
  xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
  xmlns="http://purl.org/rss/1.0/"
> 
<!--
    properties:
    - textinput in rdf:seq (illegal afaik)
    - funky person parsing test case (<dc:creator>)
    - entities in <dc:rights> and <dc:creator>
-->

  <channel rdf:about="http://Don.antville.org/">
    <title>Irgendwas ist ja immer</title>
    <link>http://Don.antville.org/</link>
    <description></description>
    <dc:publisher>antville.org</dc:publisher>
    <dc:date>2006-03-18T01:48:42Z</dc:date>
    <dc:language>en</dc:language>
    <sy:updatePeriod>hourly</sy:updatePeriod>
    <sy:updateFrequency>1</sy:updateFrequency>
    <sy:updateBase>2000-01-01T00:00:00Z</sy:updateBase>
    <items>
      <rdf:Seq>
        <rdf:li rdf:resource="http://Don.antville.org/stories/1353880/" />
         <textinput rdf:resource="http://Don.antville.org/search" />
      </rdf:Seq>
    </items>
  </channel>

  <item rdf:about="http://Don.antville.org/stories/1353880/">
    <title>Haben Sie schon an Ihre Beerdigung...</title> 
    <link>http://Don.antville.org/stories/1353880/</link>
    <description>Haben Sie schon an Ihre Beerdigung gedacht? 
Ich habe zu meinen Kindern gesagt, dass ich keine &amp;ouml;ffentliche Beerdigung will. Aus Angst vor den &amp;raquo;Jacob Sisters&amp;laquo;! Mit ihren komischen Pudeln zerst&amp;ouml;ren sie doch jede Atmosph&amp;auml;re. Die sind auch bei Moshammer aufgetaucht! Deshalb: keine &amp;ouml;ffentliche Beerdigung aus Angst vor den &amp;raquo;Jacob Sisters&amp;laquo;. Das k&amp;ouml;nnen Sie ruhig schreiben.

Wundersch&amp;ouml;nes, sehr trauriges Interview mit einem sterbenskranken Rudi Carell...</description>
    <dc:publisher>antville.org</dc:publisher>
    <dc:creator>DonDahlmann (mailto:&amp;#100;&amp;#111;&amp;#110;&amp;#100;&amp;#97;&amp;#104;&amp;#108;&amp;#109;&amp;#97;&amp;#110;&amp;#110;&amp;#64;&amp;#103;&amp;#109;&amp;#97;&amp;#105;&amp;#108;&amp;#46;&amp;#99;&amp;#111;&amp;#109;)</dc:creator>
    
    <dc:rights>Copyright &#169; 2006 DonDahlmann</dc:rights>
    <dc:date>2006-03-18T01:17:35Z</dc:date>
  </item> 
 
   <textinput rdf:about="http://Don.antville.org/search">
      <title>find</title>
      <description>Search this site:</description>
      <name>q</name>
      <link>http://Don.antville.org/search</link>
   </textinput>
</rdf:RDF>
