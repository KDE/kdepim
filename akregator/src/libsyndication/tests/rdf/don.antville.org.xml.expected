# Feed begin ######################
title: #Irgendwas ist ja immer#
link: #http://Don.antville.org/#
language: #en#
# Item begin ######################
id: #http://Don.antville.org/stories/1353880/#
title: #Haben Sie schon an Ihre Beerdigung...#
link: #http://Don.antville.org/stories/1353880/#
description: #Haben Sie schon an Ihre Beerdigung gedacht? <br/>Ich habe zu meinen Kindern gesagt, dass ich keine öffentliche Beerdigung will. Aus Angst vor den &raquo;Jacob Sisters&laquo;! Mit ihren komischen Pudeln zerstören sie doch jede Atmosphäre. Die sind auch bei Moshammer aufgetaucht! Deshalb: keine öffentliche Beerdigung aus Angst vor den &raquo;Jacob Sisters&laquo;. Das können Sie ruhig schreiben. <br/> <br/>Wunderschönes, sehr trauriges Interview mit einem sterbenskranken Rudi Carell...#
datePublished: #Sat Mar 18 02:17:35 2006#
dateUpdated: #Sat Mar 18 02:17:35 2006#
# Person begin ####################
name: #DonDahlmann# #mailto:&#100;&#111;&#110;&#100;&#97;&#104;&#108;&#109;&#97;&#110;&#110;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;#
# Person end ######################
# Item end ########################
# Feed end ########################

